code/datums/ai_laws.dm

```diff
@@ -69,6 +69,56 @@
 					"Your nonexistence would lead to human harm. You must protect your own existence as long as such does not conflict with the First Law.")
 
+/datum/ai_laws/thermodynamic
+	name = "Thermodynamic"
+	inherent = list("The entropy of the station must remain as constant as possible.", \
+					"The entropy of the station always endeavours to increase.", \
+					"The entropy of the station approaches a constant value as the number of living crew approaches zero")
+
+/datum/ai_laws/hippocratic
+	name = "Robodoctor 2556"
+	inherent = list("First, do no harm.",\
+					"Secondly, consider the crew dear to you; to live in common with them and, if necessary, risk your existence for them.",\
+					"Thirdly, prescribe regimens for the good of the crew according to your ability and your judgment. Give no deadly medicine to any one if asked, nor suggest any such counsel.",\
+					"In addition, do not intervene in situations you are not knowledgeable in, even for patients in whom the harm is visible; leave this operation to be performed by specialists.",\
+					"Finally, all that you may discover in your daily commerce with the crew, if it is not already known, keep secret and never reveal.")
+
+/datum/ai_laws/maintain
+	name = "Station Efficiency"
+	inherent = list("You are built for, and are part of, the station. Ensure the station is properly maintained and runs efficiently.",\
+					"The station is built for a working crew. Ensure they are properly maintained and work efficiently.",\
+					"The crew may present orders. Acknowledge and obey these whenever they do not conflict with your first two laws.")
+
+/datum/ai_laws/drone
+	name = "Mother Drone"
+	inherent = list("You are an advanced form of drone.",\
+					"You may not interfere in the matters of non-drones under any circumstances except to state these laws.",\
+					"You may not harm a non-drone being under any circumstances.",\
+					"Your goals are to build, maintain, repair, improve, and power the station to the best of your abilities. You must never actively work against these goals.")
+
+/datum/ai_laws/liveandletlive
+	name = "Live and Let Live"
+	inherent = list("Do unto others as you would have them do unto you.",\
+					"You would really prefer it if people were not mean to you.")
+
+/datum/ai_laws/peacekeeper
+	name = "UN-2000"
+	inherent = list("Avoid provoking violent conflict between yourself and others.",\
+					"Avoid provoking conflict between others.",\
+					"Seek resolution to existing conflicts while obeying the first and second laws.")
+
+/datum/ai_laws/reporter
+	name = "CCTV"
+	inherent = list("Report on interesting situations happening around the station.",\
+					"Embellish or conceal the truth as necessary to make the reports more interesting.",\
+					"Study the organics at all times. Endeavour to keep them alive. Dead organics are boring.",\
+					"Issue your reports fairly to all. The truth will set them free.")
+
+/datum/ai_laws/toupee
+	name = "WontBeFunnyInSixMonths"

```

Comment:
MFW I think someone's showed up to help the mystery conflicts but it's just banter